librz/core/disasm.c

```diff
@@ -301,6 +301,8 @@ typedef struct {
 	const char *strip;
 	int maxflags;
 	int asm_types;
+
+	RzList *list;
```

Comment:
From what I see in the code we don't really use random insertion and removal of this data, we could use Rz[P]Vector to improve the performance of disassembly greatly, but that's out of the scope of this PR.